Dockerfile

```
@@ -1,10 +1,15 @@
-FROM ghcr.io/astral-sh/uv:0.6-python3.12-bookworm
+# use python 3.12 as a base image
+FROM docker.io/python:3.12-alpine
+# use the latest version of uv, independently of the python version
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 
 WORKDIR /app
 
-COPY pyproject.toml uv.lock ./
+# copy the requirements and install them
+COPY pyproject.toml uv.lock .
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)